packages/story-editor/src/elements/video/editMenu.js

```diff
@@ -19,17 +19,28 @@
  */
 import useVideoTrim from '../../components/videoTrim/useVideoTrim';
 import VideoTrimmer from '../../components/videoTrim/videoTrimmer';
+import StoryPropTypes from '../../types';
 
-function VideoEditMenu() {
-  const { isTrimMode } = useVideoTrim(({ state: { isTrimMode } }) => ({
-    isTrimMode,
-  }));
+function VideoEditMenu({ element: { resource } }) {
+  const { isTrimMode, shouldFetchResource } = useVideoTrim(
+    ({ state: { isTrimMode }, actions: { shouldFetchResource } }) => ({
+      isTrimMode,
+      shouldFetchResource,
+    })
+  );
 
   if (!isTrimMode) {
     return false;
   }
-
+  // Don't display trimmer if the original resource isn't loaded.
+  if (shouldFetchResource(resource?.trimData || {})) {
+    return false;
```

Comment:
Since the `editMenu` is using the general values from the provider and not values per the selected video element then unless we hide it for the time of fetching the original resource, it displays the previous value.

One thing that's still a todo here is to reset the values in case there is an existing value but the new video doesn't have origin resource. So entering trim mode for a video that sets the trim values and then entering trim mode for a video that doesn't have an origin resource will currently display the offsets of the previous video instead of "clean" offsets.

I'm thinking that perhaps it would make sense to reset the start and end offsets and the `originalResource` whenever we enter trim mode.